submissions/SofiiaTrokhymchuk/pop-up-task/css/style.css

```diff
@@ -0,0 +1,217 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style-type: none;
+}
+
+body {
+    font-family: Verdana, Tahoma, sans-serif;
+    color: #212121;
+    background: #e6e6e6;
+    font-size: 1.2rem;
+    height: 100vh;
+}
+
+a:link,
```

Comment:
Avoid styling by HTML tag
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css